tests/integration/docker/cmd_docker_devfile_push_test.go

```diff
@@ -0,0 +1,81 @@
+package docker
```

Comment:
how do you feel about all devfile tests being devfile dir? and then separated out to `tests/integration/devfile/docker` and `tests/integration/devfile/kube`

Reply:
Makes sense, fixed.